rust/utils/run-locally/src/main.rs

```diff
@@ -412,7 +403,6 @@ fn main() -> ExitCode {
     while !SHUTDOWN.load(Ordering::Relaxed) {
         if config.ci_mode {
             // for CI we have to look for the end condition.
-            // if termination_invariants_met(&config, starting_relayer_balance)
             if termination_invariants_met(
```

Comment:
after https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3870 is merged, it may be nice to follow up here to make sure that we hit the hook indexing code path

Reply:
will do, but I did test manually with the watermark cursor disabled and it worked